fastchat/modules/paths.py

```diff
@@ -0,0 +1,28 @@
+import os
```

Comment:
Why is this file required? There are many hardcoded strings. Could you set them as arguments or environment variables?

Reply:
I try to borrow the style of [stable-diffusion-webui](https://github.com/AUTOMATIC1111/stable-diffusion-webui) to introduce dependent libraries:

https://github.com/AUTOMATIC1111/stable-diffusion-webui/blob/89f9faa63388756314e8a1d96cf86bf5e0663045/modules/paths.py#L21-L42

The way of text-generation-web-ui:

https://github.com/oobabooga/text-generation-webui/blob/78b2478d9c6cfbe7582f5a7982797bb23a41a1dd/modules/GPTQ_loader.py#L14

The former introduces more checks, and I think this way is more conducive to managing dependent libraries.